core-bundle/src/Resources/config/services.yml

```diff
@@ -815,3 +815,15 @@ services:
             - '@request_stack'
             - '@contao.framework'
             - '@contao.routing.scope_matcher'
+
+    contao.util.simple_token_parser:
+        class: Contao\CoreBundle\Util\SimpleTokenParser
+        public: true
+        arguments:
+            - '@contao.util.simple_token_parser.expression_language'
```

Comment:
Should we tag the logger to use the correct channel?

Reply:
You mean this, right?
```
        tags:
            - { name: monolog.logger, channel: contao }
```

Reply:
The `BrokenLinkCheckerSubscriber` and the `SearchIndexSubscriber` also implement the `LoggerAwareInterface`. Should we tag them as well?

https://github.com/contao/contao/blob/5c6387596f18d6e2ae0fbf7ed39483a149d493c3/core-bundle/src/Crawl/Escargot/Subscriber/BrokenLinkCheckerSubscriber.php#L30